supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -81,7 +81,7 @@ _Specify the system architecture of your cluster, such as `x86_64` or `aarch64`.
 
 [[ack]]
 ==== image:images/no.png[no] ACK (noun)
-*Description*: Do not use _ACK_ as acronym for "acknowledgement" in general. When writing about the acknowledgement flag ("ACK flag") in a TCP packet, use "ACK flag".
+*Description*: Do not use _ACK_ as acronym for "acknowledgment" in general. When writing about the acknowledgment flag ("ACK flag") in a TCP packet, use "ACK flag".
```

Comment:
```suggestion
*Description*: Do not use _ACK_ as an abbreviation for "acknowledgment". When writing about the acknowledgment flag ("ACK flag") in a TCP packet, use "ACK flag".
```
`ACK` is not an acronym, according to [M-W Dictionary's definition](https://www.merriam-webster.com/dictionary/acronym).

Reply:
Nice catch. Thank you. Updated.